devtools-ci-index.yaml

```diff
@@ -563,6 +563,35 @@
               current-parameters: true
 
 - job:
+    name: 'eclipse-che-promotion'
+
+    parameters:
+        - string:
+            name: DOCKER_REPO
+            description: "Full docker location with organization/repo:tag"
```

Comment:
This is the stage repository right? And this is the Docker image name not the repo name? In this case I think you should change the name of the parameter to something less confusing (e.g. CHE_SERVER_STAGE_DOCKER_IMAGE). And then change PRDO_DOCKER too (e.g. CHE_SERVER_PROD_DOCKER_IMAGE)